components/avatar.css

```diff
@@ -0,0 +1,54 @@
+.avatar__example-demo-container{
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
+
+.img-round{
+    border-radius: 50%;
```

Comment:
You can use object-fit: cover property, then your avatar pics will look better.

Reply:
Thanks for this suggestion. This really helped.